src/appellate/appellate.js

```diff
@@ -72,22 +72,37 @@ AppellateDelegate.prototype.handleCaseSelectionPage = async function () {
     this.pacer_case_id = APPELLATE.getCaseIdFromCaseSelection();
     await saveCaseIdinTabStorage({ tabId: this.tabId }, this.pacer_case_id);
 
-    this.recap.getAvailabilityForDocket(this.court, this.pacer_case_id, (result) => {
-      if (result.count === 0) {
-        console.warn('RECAP: Zero results found for docket lookup.');
-      } else if (result.count > 1) {
-        console.error(`RECAP: More than one result found for docket lookup. Found ${result.count}`);
-      } else {
-        if (result.results) {
+    let dataTable = APPELLATE.getTableWithDataFromCaseSelection();
+    let anchors = dataTable.querySelectorAll('a');
+    let districtLink = anchors[anchors.length-1]
+    let districtLinkData = APPELLATE.getDatafromDistrictLinkUrl(districtLink.href);
+
+    this.recap.getAvailabilityForDocket(this.court, this.pacer_case_id, null, (result) => {
+        if (result.count === 1 && result.results) {
           PACER.removeBanners();
+          
           const footer = document.querySelector('div.noprint:last-of-type');
           const div = document.createElement('div');
           div.classList.add('recap-banner');
           div.appendChild(recapAlertButton(this.court, this.pacer_case_id, true));
-          footer.before(recapBanner(result.results[0]));
           footer.before(div);
+
+          const rIcon = APPELLATE.makeRButtonForCases(result.results[0].absolute_url);
+          const appellateLink = anchors[0];
+          rIcon.insertAfter(appellateLink);
+        }else{
```

Comment:
I guess we should add prettier to our CI system somehow, but shouldn't this be:

```suggestion
        } else {
```

To be clear, I don't really care about this one instance, but it'd be good to be confident that our code is always clean and tidy. Some things prettier does are more important too, like adding semicolons?

Reply:
You're right about the styling here.